business/openid_auth.go

```diff
@@ -247,86 +272,150 @@ func GetOpenIdMetadata() (*OpenIdMetadata, error) {
 		return cachedOpenIdMetadata, nil
 	}
 
-	cfg := config.Get().Auth.OpenId
+	fetchedMetadata, fetchError, _ := openIdFlightGroup.Do("metadata", func() (interface{}, error) {
+		cfg := config.Get().Auth.OpenId
 
-	// Remove trailing slash from issuer URI, if needed
-	trimmedIssuerUri := strings.TrimRight(cfg.IssuerUri, "/")
+		// Remove trailing slash from issuer URI, if needed
```

Comment:
I think your editor did something bad here :)

Reply:
Is it because of the extra indentation?
If so, I think it's fine. All this code is now enclosed in an anonymous function. on line 275 (two lines before this one):

`fetchedMetadata, fetchError, _ := openIdFlightGroup.Do("metadata", func() (interface{}, error) {`

or is it something else?

Reply:
Yeah, the indentation is mixed with tabs/spaces.

Reply:
Hmmmmmm... I reformatted, but it does not detect any changes...

I consistently see tabs on all changed lines... the added indentation are tabs and the existent indentation are tabs too :/ Even here in GitHub when selecting text the browser is highlighting tabs...

Can you check? I would think these formatting issues would be detected by the linters in the CI.

Reply:
I checked out the PR and the formatting is fine for me.  I think this comment can be resolved.  If this is the only issue then we can approve.